jlm/llvm/opt/alias-analyses/PointerObjectSet.cpp

```diff
@@ -1360,7 +1386,7 @@ PointerObjectConstraintSet::NormalizeConstraints()
   return reduction;
 }
 
-template<typename Worklist, bool EnableOnlineCycleDetection>
+template<typename Worklist, bool EnableOnlineCycleDetection, bool EnableHybridCycleDetection>
```

Comment:
As far as I understand it, it should never happen that both, `EnableOnlineCycleDetection` and `EnableHybridCycleDetection`, are enabled at the same time. Maybe we can add a `static_assert<>()` that tries to check for that?